doc/Type/Seq.pod6

```diff
@@ -8,10 +8,20 @@
 
 A C<Seq> represents anything that can lazily produce a sequence of values. A
 C<Seq> is born in a state where iterating it will consume the values. However,
-calling .cache on a Seq will return a List that is still lazy, but stores the
-generated values for later access. The same is true when assigning a Seq to an
-array.
+calling C<.cache> on a C<Seq> will return a List that is still lazy, but stores the
+generated values for later access. Assigning a C<Seq> to an array consumes the C<Seq>;
```

Comment:
Maybe a "however" before Assigning?

Reply:
Thanks for reviewing, I just addressed your comments, please check it out. Thanks.